### ISSUE TITLE
replace directives during go install

### ISSUE THREAD

**Issue by tcurdt** [apricote/hcloud-upload-image#57]
```
$ go install github.com/apricote/hcloud-upload-image@latest
go: downloading github.com/apricote/hcloud-upload-image v0.3.0
go: github.com/apricote/hcloud-upload-image@latest (in github.com/apricote/hcloud-upload-image@v0.3.0):
	The go.mod file for the module providing named packages contains one or
	more replace directives. It must not contain directives that would cause
	it to be interpreted differently than if it were the main module.
```

This is because the latest release still has the replace directive.
What works is `go install github.com/apricote/hcloud-upload-image@main`.

Maybe time to create a new release without the replace directive?

**Comment by apricote** [comment]
I released 0.3.1. Can you try the install command again?

**Comment by tcurdt** [comment]
works

```
$ go install github.com/apricote/hcloud-upload-image@latest
go: downloading github.com/apricote/hcloud-upload-image v0.3.1
go: downloading github.com/hetznercloud/hcloud-go/v2 v2.17.0
go: downloading golang.org/x/crypto v0.30.0
go: downloading golang.org/x/text v0.21.0
go: downloading golang.org/x/sys v0.28.0
$
```
